dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -9,6 +9,8 @@
 - show_lesson_plans = !is_student
 - no_lesson_plans = !pdfs && !has_pegasus_lesson_plan
 
+= stylesheet_link_tag '/blockly/video-js/video-js', media: 'all'
```

Comment:
This will load video-js.css regardless of whether a video even exists on this page, and regardless of whether the browser requires the fallback player, which is doubly inefficient.

Reply:
We can presumably at least easily make this conditional on there being a video, right? I don't know enough about when we use the fallback player to know how easy it is to avoid the second inefficiency

Reply:
It would be a little gross, but we could also have the fallback video player itself load this css.